command/unixOSCommand.go

```diff
@@ -0,0 +1,85 @@
+package command
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	"github.com/alessio/shellescape"
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+const (
+	linuxTempDir = "/tmp"
+)
+
+var _ osCommand = (*unixOSCommand)(nil)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is this used for ?

Reply:
This is the Go way to check a struct implement an interface.

Reply:
Nice, thank you !